pkgs/top-level/all-packages.nix

```
@@ -26597,6 +26606,26 @@ with pkgs;
 
   thttpd = callPackage ../servers/http/thttpd { };
 
+  stalwart-mail_0_6 = stalwart-mail.overrideAttrs (old: rec {
+    pname = "stalwart-mail_0_6";
+    version = "0.6.0";
+    src = fetchFromGitHub {
+      owner = "stalwartlabs";
+      repo = "mail-server";
+      rev = "v${version}";
+      hash = "sha256-OHwUWSUW6ovLQTxnuUrolQGhxbhp4YqKSH+ZTpe2WXc=";
+      fetchSubmodules = true;
+    };
+    cargoDeps = old.cargoDeps.overrideAttrs (_: {
+      inherit src;
```

Tho i am fan here of the overriding, i am not sure if here is kind of a bug somewhere.
When building version 0.6.0 i have the following log

```
stalwart-mail_0_6> unpacking source archive /nix/store/y6ncfqcyms9g4n6qhakc22zfqjlgqrrc-stalwart-mail-0.8.0-vendor.tar.gz
```
which sounds not correct to me.

```suggestion
      inherit src;
      name = "${pname}-${version}";
```

When we override the name, that in no longer the case.